src/main/java/ua/realestate/management/security/config/WebMvcConfig.java

```diff
@@ -0,0 +1,21 @@
+package ua.realestate.management.security.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebMvcConfig implements WebMvcConfigurer {
+    private final long MAX_AGE_SECS = 1200;
+
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
+        registry.addMapping("/**")
+                .allowedOrigins("*")
+                .allowedOrigins("http://localhost:8080", "http://localhost:4200")
+                .allowedMethods("HEAD", "OPTIONS", "GET", "POST", "PUT", "PATCH", "DELETE")
```

Comment:
If you don't plan to use all of the methods - leave only those, which you were really going to implement

Reply:
Fixed